<?xml version="1.0"?>
<robot name="Robot1">
  <link name="base_link">
    <visual>
      <geometry>
        <box size="0.2 .3 .1"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0.05"/>
      <material name="white">
        <color rgba="1 1 1 0.1"/>
      </material>
    </visual>
  </link>

  <link name="wheel_1">
    <visual>simple_robot.png
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.05" radius="0.05"/>
      </geometry>
      <material name="red">
        <color rgba="0.8 0 0 1.0"/>
      </material>
    </visual>
  </link>
  <joint name="base_to_wheel1" type="continuous">
    <parent link="base_link"/>
    <child link="wheel_1"/>
    <axis xyz="0 0 1"/>
    <origin rpy="0 1.57 0" xyz="0.1 0.1 0"/>
  </joint>

  <link name="wheel_2">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.05" radius="0.05"/>
      </geometry>
      <material name="red"/>
    </visual>
  </link>
  <joint name="base_to_wheel2" type="continuous">
    <parent link="base_link"/>
    <child link="wheel_2"/>
    <axis xyz="0 0 1"/>
    <origin rpy="0 1.57 0" xyz="-0.1 0.1 0"/>
  </joint>

  <link name="wheel_3">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.05" radius="0.05"/>
      </geometry>
      <material name="red"/>
    </visual>
  </link>
  <joint name="base_to_wheel3" type="continuous">
    <parent link="base_link"/>
    <child link="wheel_3"/>
    <axis xyz="0 0 1"/>
    <origin rpy="0 1.57 0" xyz="0.1 -0.1 0"/>
  </joint>

  <link name="wheel_4">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.05" radius="0.05"/>
      </geometry>
      <material name="red"/>
    </visual>
  </link>
  <joint name="base_to_wheel4" type="continuous">
    <parent link="base_link"/>
    <child link="wheel_4"/>
    <axis xyz="0 0 1"/>
    <origin rpy="0 1.57 0" xyz="-0.1 -0.1 0"/>
  </joint>
</robot>
